src/alpaca_eval/decoders/jinachat.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import multiprocessing
+from functools import partial
+from typing import Sequence
+import requests
+import json
+import os
+import time
+from .openai import _prompt_to_chatml
+from .. import utils
+
+__all__ = ["jina_chat_completions"]
+
+
+def jina_chat_completions(
+    prompts: Sequence[str],
+) -> dict[str, list]:
+    """Get jina chat completions for the given prompts. Allows additional parameters such as tokens to avoid and
+    tokens to favor.
+
+    Parameters
+    ----------
+    prompts : list of str
+        Prompts to get completions for.
+    """
+    n_examples = len(prompts)
+    api_key = os.environ.get('JINA_CHAT_API_KEY')
+
+    if n_examples == 0:
+        logging.info("No samples to annotate.")
+        return {}
+    else:
+        logging.info(f"Using `jina_chat_completions` on {n_examples} prompts.")
+
+    prompts = [_prompt_to_chatml(prompt.strip()) for prompt in prompts]
+    num_processes = min(multiprocessing.cpu_count(), 4)
+    with utils.Timer() as t:
+        with multiprocessing.Pool(processes=num_processes) as pool:
+            print(f"Number of processes: {pool._processes}")
+            get_chat_completion_with_key = partial(_get_chat_completion, api_key)
+            completions = pool.map(get_chat_completion_with_key, prompts)
+
+    logging.info(f"Completed {n_examples} examples in {t}.")
+
+    # refer to https://chat.jina.ai/billing
+    price = 0
```

Comment:
price is not 0. Pleasier either:
- use `[np.nan] * len(completions)` to say that the price is not given
- (better) use the estimated price which seems to be approximately `[0 if len(c) < 100 else 0.08 for c in completions]`

Reply:
Following the standard package, I calculate pricing in the following way:

if msg tokens is more than 300, price is 0.08
otherwise 0